h11/_readers.py

```diff
@@ -32,9 +54,9 @@
 obs_fold_re = re.compile(br"[ \t]+")
 
 
-def _obsolete_line_fold(lines):
+def _obsolete_line_fold(lines: Iterable[bytes]) -> Generator[bytes, None, None]:
```

Comment:
Is `Generator` relevant to this API since we're not using `.send()`? Maybe should be `Iterable[bytes]`?

Reply:
Yea, changed. Thanks